pom.xml

```diff
@@ -10,6 +10,7 @@
         <module>data</module>
         <module>management</module>
         <module>account</module>
+        <module>samples</module>
```

Comment:
Inside the Eclipse IDE the effect of adding the sample module allows to define run-configurations that are needed to start the samples from within the IDE